services/job_chat/tests/test_qualitative_old_prompt.py

```diff
@@ -0,0 +1,194 @@
+import pytest
+import json
+from .test_utils import call_job_chat_service, make_service_input, print_response_details
+
+
+def test_basic_input():
```

Comment:
Are the qualitative tests for the old prompt of any value? I think a couple of basic pass/fail, just to ensure the prompt is loaded at the right time, feels like enough. Because right now we're basically quality testing legacy code, which seems of little value